pages/common/samtools.md

```diff
@@ -0,0 +1,36 @@
+# samtools
+
+> Tools for handling high-throughput sequencing (genomics) data.
+> Used for reading/writing/editing/indexing/viewing of data in SAM/BAM/CRAM format.
+
+- Convert a SAM input file to BAM stream and save to file:
+
+`samtools view -S -b {{input.sam}} > {{output.bam}}`
+
+- Take input from stdin (-) and print the SAM header and any reads overlapping a specific region to stdout:
+
+`{{other_command}} | samtools view -h - chromosome:start-end`
+
+- Sort file and save to BAM (the output format is automatically determined from the output file's extension):
+
+`samtools sort {{input}} -o {{output.bam}}`
+
+- Index a sorted BAM file (creates {{sorted_input.bai}}):
```

Comment:
Oh I see, nope! That's meant to be `bai` :) Short for for _bam index_. Though actually you are correct that there's a typo - it should be `sorted_input.bam.bai` now that we have the input file as `sorted_input.bam`